testsuite/python/polymer_linear.py

```diff
@@ -21,7 +21,7 @@
 import espressomd.shapes
 
 
-class PolymerPositions(ut.TestCase):
+class LinearPolymerPolymerPositions(ut.TestCase):
```

Comment:
`LinearPolymerPosition`?

Reply:
I think we the name of the testclass is actually not too important if the class has a docstring.

Reply:
it's a typo nonetheless, I will fix it